Backpack/src/main/java/net/skyscanner/backpack/checkbox/BpkCheckbox.kt

```diff
@@ -30,27 +30,33 @@ open class BpkCheckbox @JvmOverloads constructor(
   }
 
   private fun initialize(attrs: AttributeSet?, defStyleAttr: Int) {
-    var defaultColor = ContextCompat.getColor(context, R.color.bpkTextSecondary)
-    var checkedColor = BpkTheme.getPrimaryColor(context)
-    var disabledColor = ContextCompat.getColor(context, R.color.bpkSkyGrayTint04)
+    var checkmarkDefaultColor = ContextCompat.getColor(context, R.color.bpkTextSecondary)
+    var checkmarkCheckedColor = BpkTheme.getPrimaryColor(context)
+    var checkmarkDisabledColor = ContextCompat.getColor(context, R.color.bpkSkyGrayTint04)
```

Comment:
Shouldn't this be the same as the text colour?

Reply:
No, if you look on the previous screenshots you notice it was always darker.

Reply:
Yes but just a little bit, and because we were not setting the text colour before. Now is a very visible difference. How does it look it in dark mode? Why can't we just use `bpkSkyGrayTint04` for the disabled text colour too?

Reply:
Done